src/components/visualizations/chart/highcharts/test/getAxisNameConfiguration.spec.ts

```diff
@@ -56,4 +58,78 @@ describe("getAxisNameConfiguration", () => {
             ],
         });
     });
+
+    it.each([
+        {
+            type: VisualizationTypes.COMBO,
+            xAxes: [{}, { opposite: true }],
+            yAxes: [{}, { opposite: true }],
+        },
+        {
+            type: VisualizationTypes.COMBO2,
+            xAxes: [{}, { opposite: true }],
+            yAxes: [{}, { opposite: true }],
+        },
+        {
+            type: VisualizationTypes.COLUMN,
+            xAxes: [{}, { opposite: true }],
+            yAxes: [{}, { opposite: true }],
+        },
+        {
+            type: VisualizationTypes.LINE,
+            xAxes: [{}, { opposite: true }],
+            yAxes: [{}, { opposite: true }],
+        },
+    ])(
+        "should return highchart axis config with rotated opposite Y axis label",
```

Comment:
If `it.each` is used, the name of the test should be parametrized by some of its parameters. If one of the test would fail, you would see in the log that second of the five tests with the same name failed which does not say much to you without looking into the source code. I would put type of the chart into the name.